modeling-cmds/src/shared.rs

```diff
@@ -131,6 +137,162 @@ pub struct AnnotationTextOptions {
     pub point_size: u32,
 }
 
+/// Parameters for defining an MBD Geometric control frame
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+#[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(export_to = "ModelingCmd.ts"))]
+pub struct AnnotationMBDControlFrame {
```

Comment:
Please rename all acronyms and initialisms as words. For example in this case it should be `AnnotationMbdControlFrame`.

This comes from the Rust API guidelines. https://rust-lang.github.io/api-guidelines/naming.html